lighthouse-core/audits/byte-efficiency/byte-efficiency-audit.js

```diff
@@ -136,6 +136,8 @@ class UnusedBytes extends Audit {
    */
   static createAuditResult(result, graph) {
     const simulatorOptions = PredictivePerf.computeRTTAndServerResponseTime(graph);
+    // TODO: calibrate multipliers, see https://github.com/GoogleChrome/lighthouse/issues/820
+    Object.assign(simulatorOptions, {cpuTaskMultiplier: 1, layoutTaskMultiplier: 1});
```

Comment:
why reset these multipliers to 1 rather than their defaults?

Reply:
well in our pre-lantern case the throttling is usually being applied by devtools, so pumping it through another 4x multiplier is sorta unfair. This shouldn't really matter once #4766 and follow-ups land

Reply:
aye true. k.